src/blocks/file-blocks/excalidraw/index.tsx

```diff
@@ -13,9 +13,7 @@ if (typeof window !== "undefined") {
   }
 }
 export default function (props: FileBlockProps) {
-  const { content, onRequestUpdateContent } = props;
-  const [appState, setAppState] = useState(null);
-  const [elements, setElements] = useState([]);
+  const { context, content, onRequestUpdateContent } = props;
```

Comment:
this should update the currently-mounted Excalidraw instance if `content` has changed; but it can wait until there is a way to observe it...
🤔 
...oh wait I just realized why the `key`s were given as `content` in these blocks — so the block updates when you look at other versions — now that I made the keys `path` this won't work — I will fix it before merging.

Reply:
also we probably want to remount blocks when you change path/version so undo state etc. doesn't carry over. I think this should be handled by adding a key in `blocks` rather than each block needing to do it.